client/client.go

```diff
@@ -337,22 +352,17 @@ func (r *NotaryRepository) Publish() error {
 	if err != nil {
 		if _, ok := err.(store.ErrMetaNotFound); ok {
 			// if the remote store return a 404 (translated into ErrMetaNotFound),
-			// the repo hasn't been initialized yet. Attempt to load it from disk.
+			// there is no trust data for yet. Attempt to load it from disk.
 			err := r.bootstrapRepo()
 			if err != nil {
-				// Repo hasn't been initialized, It must be initialized before
-				// it can be published. Return an error and let caller determine
-				// what it wants to do.
-				logrus.Debug(err.Error())
```

Comment:
Fixed in https://github.com/cyli/notary/commit/aaf45a9ccec53bbcfd9c88b5032f32bf8c64cfcd